cmd/flow-dps-indexer/main.go

```diff
@@ -78,6 +79,7 @@ func run() int {
 	pflag.BoolVarP(&flagIndexEvents, "index-events", "e", false, "index events")
 	pflag.BoolVarP(&flagIndexHeader, "index-headers", "h", false, "index headers")
 	pflag.BoolVarP(&flagIndexPayloads, "index-payloads", "p", false, "index payloads")
+	pflag.BoolVarP(&flagIndexResults, "index-results", "r", false, "index transaction results")
```

Comment:
You wanted to convert this to `pflag.BoolVar` without a short flag :D

Reply:
hehe thanks